core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -546,62 +546,28 @@ describe('Trace Elements gatherer - Animated Elements', () => {
         height: 161,
       },
     };
-    const compositedNodeData = {
-      traceEventType: 'animation',
-      devtoolsNodePath: '1,HTML,1,BODY,1,DIV',
-      selector: 'body > div#composited-boi',
-      nodeLabel: 'div',
-      snippet: '<div id="composited-boi">',
-      boundingRect: {
-        top: 169,
-        bottom: 269,
-        left: 8,
-        right: 108,
-        width: 100,
-        height: 100,
-      },
-    };
     const connectionStub = new Connection();
     connectionStub.sendCommand = createMockSendCommandFn()
       // LCP node
       .mockResponse('DOM.resolveNode', {object: {objectId: 1}})
       .mockResponse('Runtime.callFunctionOn', {result: {value: LCPNodeData}})
       // Animated node
       .mockResponse('DOM.resolveNode', {object: {objectId: 5}})
-      .mockResponse('Runtime.callFunctionOn', {result: {value: animationNodeData}})
-      // Composited node
-      .mockResponse('DOM.resolveNode', {object: {objectId: 7}})
-      .mockResponse('Runtime.callFunctionOn', {result: {value: compositedNodeData}});
+      .mockResponse('Runtime.callFunctionOn', {result: {value: animationNodeData}});
 
     const driver = new Driver(connectionStub);
     const gatherer = new TraceElementsGatherer();
     gatherer.animationIdToName.set('2', 'alpha');
     gatherer.animationIdToName.set('3', 'beta');
-    gatherer.animationIdToName.set('4', 'gamma');
 
     const result = await gatherer._getArtifact({driver, computedCache: new Map()}, animationTrace);
 
-    expect(result).toEqual([
-      {
-        ...LCPNodeData,
-        nodeId: 7,
-      },
-      {
-        ...animationNodeData,
-        animations: [
-          {failureReasonsMask: 8224, unsupportedProperties: ['width']},
-          {name: 'alpha', failureReasonsMask: 8224, unsupportedProperties: ['height']},
-          {name: 'beta', failureReasonsMask: 8224, unsupportedProperties: ['background-color']},
```

Comment:
note: background-color is now supported (sometime shortly after this audit was made), but now `font-size` replaces it in the unsupported reasons? shrug